calendars/numpy.yaml

```
@@ -37,25 +37,6 @@ events:
         days: 14
       until: 2025-01-01 00:00:00 +00:00
 
-  - summary: NumPy Documentation Team meeting (Europe/Africa/Asia)
-    description: |
-      A fortnightly meeting to discuss everything related
-      to the official NumPy documentation. Everyone is welcome to attend
-      and contribute to a conversation.
-
-      To add to the meeting agenda the topics you'd like to discuss,
-      follow the link: https://hackmd.io/oB_boakvRqKR-_2jRV-Qjg
-
-      Join us via Zoom: https://numfocus-org.zoom.us/j/85016474448?pwd=TWEvaWJ1SklyVEpwNXUrcHV1YmFJQT09
-    begin: 2022-11-07 12:00:00 +00:00
-    end: 2022-11-07 13:00:00 +00:00
-    url: https://numfocus-org.zoom.us/j/85016474448?pwd=TWEvaWJ1SklyVEpwNXUrcHV1YmFJQT09
-    repeat:
-      interval:
-        # seconds, minutes, hours, days, weeks, months, years
-        days: 28
-      until: 2025-01-01 00:00:00 +00:00
-
   - summary: NumPy Documentation Team meeting (Americas/Oceania)
```

since there is only one timeslot now, maybe the zone name should be dropped now?
```suggestion
  - summary: NumPy Documentation Team meeting
```

---

@Mukulikaa What are your thoughts on this?

---

I think we can drop it for now and add it back later if/when we have another time slot.